### ISSUE TITLE
Make it so there isnt a friccin enemy base right next to frozen forest

### ISSUE THREAD

**Issue by Colonizor48** [Anuken/Mindustry-Suggestions#1003]
**Describe what you would like changed, and why.**

Make it so there isnt a friccin enemy base right next to frozen forest so i dont get attacked by a friccin zenith when all i have to defend are mabye a few scatters.

**Describe the changes you want to propose. Include possible alternatives.**
Just remove the enemy base next to stained mountans. And any enemy bases next to any other normal(non procedual) campaign maps

**Comment by Bohaska** [comment]
Maybe a better way will be to limit the frequency of attacks to frozen forest and stained mountains.

**Comment by fredrik-krokedil** [comment]
...or make the enemy bases bordering on the campaign maps way easier to defeat, as a sort of introduction to the concept?

**Comment by darloth** [comment]
> ...or make the enemy bases bordering on the campaign maps way easier to defeat, as a sort of introduction to the concept?

This would probably be good.  I recently got to titanium and metaglass (T2 units) again in this new version and decided the time was finally right to kick out that enemy base next to frozen forest so they'd stop escalating until it inevitably dies!

Thorium walls.  Huge waves of giant purple death-spiders.  Map 3/4 full of self-sufficient red bases - that get repaired by a core drone in record time.  Maybe if I was a truly excellent player I could manage this, but... I am not.  I did some damage but of course that means nothing as it gets repaired in a minute or two.

Instead it would be lovely to be able to pull out a win at this point even if you're only okay at the game and don't have a bunch of schematics ready to go :)
